auth0/src/test/java/com/auth0/android/authentication/storage/SecureCredentialsManagerTest.kt

```diff
@@ -341,7 +330,7 @@ public class SecureCredentialsManagerTest {
     @Test
     public fun shouldNotThrowOnSaveIfCredentialsHaveIdTokenAndExpiresIn() {
         val credentials: Credentials =
-            CredentialsMock("idToken", null, "type", "refreshToken", Date(), "scope")
+            CredentialsMock("idToken", "", "type", "refreshToken", Date(), "scope")
```

Comment:
this case will never happen again, as Credentials will always have these properties passed as empty here. I'd remove this test case.